javascript/utils.js

```diff
@@ -77,10 +77,15 @@ var calcfilesize = function(bytes){
   return  bytes + " " + unit[index];
 };
 
+var supportImage = function(htmlToParse) {
+  var regexImg = /(img.*src=")\/([\w\/.\-\?=:;&]+")/g;
```

Comment:
Obviously, thanks for cathing this one xD I don't write Regex often enough I guess